styles/about.css

```diff
@@ -0,0 +1,682 @@
+  font-size: 15px;
+  width: 400px;
+  text-align: center;
+  line-height: 16px;
+  font-family: sans-serif, 'lato';
+}
+
+.mobile-image {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  align-self: center;
+}
+
+.text {
+  display: flex;
+  justify-content: center;
+  margin-top: 35px;
+}
+
+.logo-pic {
+  width: 70px;
+  height: 70px;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-self: center;
+  margin-top: 8px;
+}
+
+.past-summits {
+  text-align: center;
+  padding: 30px;
+  background: #fff;
+}
+
+.past-summits h3 {
+  font-size: 12px;
+}
+
+.past-summits h3::after {
+  display: block;
+  height: 2.16px;
+  background-color: #ec5242;
+  content: " ";
+  width: 50.2px;
+  margin: 0 auto;
+  margin-top: 5.08px;
+}
+
+.past-summits p {
+  font-family: sans-serif, 'lato';
+  line-height: 16px;
+  font-size: 13px;
+}
+
+.Summit-examples {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+}
+
+.Summit-examples img {
+  display: flex;
+  width: 350px;
+  height: 180px;
+  margin: 0 auto;
+}
+
+#d-down {
+  margin-top: 8px;
+}
+
+.partner {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #272a31;
+  color: #fff;
+  margin-top: 10px;
+  padding: 40px;
+}
+
+.footer-mobile {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.footer-mobile h2 {
+  font-size: 15px;
+  justify-content: center;
+  position: relative;
+  bottom: 20px;
+  text-align: center;
+}
+
+.footer-mobile h2::after {
+  display: block;
+  height: 2.16px;
+  background-color: #ec5242;
+  content: " ";
+  width: 50.2px;
+  margin: 0 auto;
+  margin-top: 10.08px;
+}
+
+.partners-mobile {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  align-items: flex-end;
+}
+
+.footer-mobile .partners-mobile {
+  display: flex;
+  text-decoration: none;
+  list-style-type: none;
+  text-align: center;
+  justify-content: center;
+  color: #d3d3d3;
+}
+
+.footer-container {
+  display: flex;
+  background: #fff;
+  color: #272a31;
+  justify-content: space-around;
+  align-items: center;
+  font-size: 10px;
+  padding: 30px;
+  text-align: center;
+}
+
+.footer-child .circle {
+  border-radius: 50%;
+  border: 3px solid #272a31;
+  color: #272a31;
+  margin-right: 3px;
+  padding: 2px;
+}
+
+.footer-child .rights {
+  font-size: 9px;
+}
+
+.footer-child .message {
+  line-height: 20px;
+  font-family: 'lato', sans-serif;
+}
+
+.footer-child {
+  padding-right: 10px;
+}
+
+.partners-mobile li {
+  width: 33%;
+}
+
+.partners-mobile .fab {
+  color: #8f8b8b;
+  font-size: 30px;
+  position: relative;
+  left: 60px;
+  top: 5px;
+}
+
+#dark,
+#lightgrey {
+  color: #b1aeae;
+  font-size: 25px;
+}
+
+/* Media screen for 560, 420, 768, 992, 1024 and above */
+
+@media only screen and (min-width: 768px) {
```

Comment:
* I suggest you apply the mobile-first approach design pattern as it a modern design pattern and makes your design easily scalable [Read more on why the mobile-first approach is good ](https://medium.com/@Vincentxia77/what-is-mobile-first-design-why-its-important-how-to-make-it-7d3cf2e29d00)

Reply:
did you look at my code, what is the difference between this line of code and my home style.css?